pkgs/terraform-linters/tflint/registry.yaml

```diff
@@ -52,6 +52,24 @@ packages:
           type: github_release
           asset: checksums.txt
           algorithm: sha256
+      - version_constraint: semver("<= 0.51.0")
+        asset: tflint_{{.OS}}_{{.Arch}}.{{.Format}}
+        format: zip
+        windows_arm_emulation: true
+        checksum:
+          type: github_release
+          asset: checksums.txt
+          algorithm: sha256
+          cosign:
+            opts:
+              - --certificate-identity-regexp
```

Comment:
Can we use `--certificate-identity`?

Reply:
Done. ae5e5d6c2b29abecfb7c3a13adfa6a66d4595380

Reply:
Oh, sure, I was just keeping the PR in scope, i.e. _only_ adding the attestations config.